canonical/cloud-init
# Add support for OpenMandriva

## Proposed Commit Message


```
Add support for OpenMandriva (https://openmandriva.org/).

Signed-off-by: Bernhard Rosenkränzer <bero@lindev.ch>
```

## Additional Context

Verified to work with an OpenMandriva Cooker image in Oracle Cloud.
Will try a few more clouds and working with extra modules; a few more patches may be necessary to make all modules work, but all the basics are looking good already with this initial patch.

## Test Steps
Install OpenMandriva in a supported cloud and make sure cloud-init created the user, brought up the network, set up ssh keys, resized the filesystem, etc.

## Checklist:
